tests/store_helpers.py

```diff
@@ -722,11 +739,12 @@ def ensure_invoice(
         invoice_id: int = 0,
         customer_id: str = "cust000",
         discount: int = 0,
-        record_type: str = "Sample",
-        internal_id: str = "savedkitten",
+        pools: Optional[List[models.Pool]] = None,
+        samples: Optional[List[models.Sample]] = None,

```

Comment:
maybe in this script also import `Pool` and `Sample` independently from `models`